cmd/webhook/main_test.go

```
@@ -0,0 +1,376 @@
+				}
+			})
+		})
+	})
+}
+
+func testHTTPEndpoint(address, endpoint string) bool {
+	return Eventually(func(g Gomega) {
+		resp, err := http.Get(fmt.Sprintf("http://%s/%s", address, endpoint))
+		g.Expect(err).NotTo(HaveOccurred())
+		g.Expect(resp.StatusCode).NotTo(Equal(http.StatusNotFound))
+		resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)